systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -363,6 +373,50 @@ public void testForUpdateValuesInConfigMap() {
         }
     }
 
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
+    public void testSendMessages() {
+        String topicName = "test-topic";
+        Oc oc = (Oc) this.kubeClient;
+        int messagesCount = 5;
+        oc.namespace();
+        String clusterName = "openshift-my-cluster";
+        oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));
```

Comment:
Couldn't we achieve the same thing using `@KafkaCluster`, which would avoid the need for the test to be `@OpenShiftOnly`, and save you from having to wait for the statefulsets etc.